kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+export const title = 'MDN Web Docs payment terms';
+const TermsPage = () => (
+    <>
+        <SubHeader title={title} />
+        <main
+            className="contributions-page"
+            role="main"
+            data-testid="terms-page"
+        >
+            <section>
+                <h2>Payment Terms</h2>
+
+                <p>Mozilla (that’s us) operates MDN Web Docs.</p>
+                <p>
+                    These Payment Terms cover your payment to MDN.{' '}
+                    <a
+                        href="https://www.mozilla.org/en-US/about/legal/terms/mozilla/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        The Websites & Communications Terms of Use
+                    </a>{' '}
+                    cover the MDN service itself. If there is any inconsistency
+                    between these terms and the Websites & Communications Terms
+                    of Use, these Payment Terms apply.
+                </p>
+
+                <h2>Payment Authorization</h2>
+                <p>
+                    You can support MDN with a one-time payment, or you can
+                    choose to schedule monthly payments. Your payments are not
+                    tax deductible.
+                </p>
+                <p>
+                    In order to sign up for monthly payments, you must sign in
+                    to MDN using a GitHub account. GitHub has its own{' '}
```

Comment:
Note @atopal This PR is @mindy just refactoring the payments terms page. She's just moving the text from one place to another but did you know we're not mentioning Google at all here? Is that something that should be a new issue? Also, we reads terms and conditions??